.github/workflows/coverage.yml

```
@@ -30,28 +30,16 @@ jobs:
       - name: Install rustfilt symbol demangler
         run: |
           cargo install rustfilt
-      - name: Rerun tests for coverage
+
+      - name: Install cargo-llvm-cov cargo command
+        run: |
+          cargo install cargo-llvm-cov --version 0.1.0-alpha.4
```

Looks like GitHub hid the previous conversation, so let's try:
```suggestion
          cargo install cargo-llvm-cov --version ^0.1.0-alpha.4
```